src/DotNet.Consolidate.Tests/DotNet.Consolidate.Tests.csproj

```diff
@@ -17,7 +17,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="xunit" Version="2.6.2" />
+    <PackageReference Include="xunit" Version="2.6.4" />
```

Comment:
Is this necessary for the change?

Reply:
No I can revert it if you want.  But it helped in my local testing.